python/dgl/contrib/dis_kvstore.py

```diff
@@ -275,17 +279,27 @@ class KVClient(object):
           2:'168.12.46.12:50051' }
     client_addr : str
         IP address of current KVClient, e.g., '168.12.23.22:50051'
+    partition_book : list of int
+        A book mapping for global ID and server ID
+    global_to_local : list of int
```

Comment:
why do we need to map global id to local id?

Reply:
Because when we meet local partition data, we need to convert it to the local id. But the ID passed to pull/push is global ID.